grpc/src/main/java/io/stargate/grpc/codec/cql/BytesCodec.java

```diff
@@ -0,0 +1,30 @@
+package io.stargate.grpc.codec.cql;
+
+import com.google.protobuf.ByteString;
+import io.grpc.Status;
+import io.grpc.StatusException;
+import io.stargate.db.schema.Column;
+import io.stargate.proto.QueryOuterClass;
+import io.stargate.proto.QueryOuterClass.Value;
+import io.stargate.proto.QueryOuterClass.Value.InnerCase;
+import java.nio.ByteBuffer;
+import org.jetbrains.annotations.NotNull;
```

Comment:
so I see that no other module except test uses the `jetbrain-annotations`, was this intentional?

Reply:
Good catch. Not sure what happened there.